docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md

```diff
@@ -266,6 +280,121 @@ az acr connected-registry show \
+
+            "$metadata": {...}
+```
+The value of the proxy_config is the base64 encoded string of the following nginx config.
```

Comment:
I guess you mean to copy/paste the nginx config below.

Reply:
Yeah, copied the wrong one. Updated the config.